### DISCUSSION TITLE
Any reason to strip response body when no-cors mode is provided to fetch?

### DISCUSSION THREAD

**Discussion by raythurnvoid** [sveltejs/kit#8097] [Q&A]
I'm using the svelte-kit's provided fetch method and calling the Microsoft token API to get a new token and call the ms-graph API; in my current configuration, I have to perform the request from Node; Microsoft checks the Origin header and will respond with an error if set. I've tried to provide fetch with mode: 'no-cors' and I've noticed that the response body is stripped away: https://github.com/sveltejs/kit/blob/ab5f1c1c34fffa91e062cd6dc431a5e99cc53097/packages/kit/src/runtime/server/fetch.js#L68

Is there any reason to remove the body response when no-cors is being used?

**Comment by Rich-Harris** [comment]
Is this happening in a `fetch` from a shared `load` function (`+page.js`) or a server-only one (`+page.server.js`)? If the first, this is intentional — the behaviour needs to be consistent between browser and server. If the second, you may have found a bug — please open an issue!

**Reply by raythurnvoid** [reply]
Actually, it's happening in the +server.js module, i'm using the fetch provided in `event: RequestEvent`

![image](https://user-images.githubusercontent.com/53383860/207629932-fc956926-6475-4848-ae96-29438345e5ed.png)

at the moment, I worked around it by using the global fetch provided by node.
